arcgis-ios-sdk-samples/Display information/Custom dictionary style/README.md

```diff
@@ -2,7 +2,7 @@
 
 Use a custom dictionary style (.stylx) to symbolize features using a variety of attribute values.
 
-![Image of custom dictionary style](image1.png)
+![Image of custom dictionary style](custom-dictionary-style.png)
```

Comment:
Are these images renames really needed for this release? Ideally only bug fixes should go in to a release branch. These renames seem more like style updates than bug fixes.

Reply:
That makes sense. 

Initially I thought it would be nice to make the most correct formatting into the release, in case there won't be another thorough README update in the near future. But I also agree that it should be addressed in a separate issue.

Do I revert those?

Reply:
Yes, please revert them.